### PR TITLE
[CDAP-6139] Fixed spacing and alignment for 'Null' and 'Trash' column

### PR DESCRIPTION
**Pull request by PGuimarais**
Fix for: https://issues.cask.co/browse/CDAP-6139

Reformatted the output schema widget

Null Column:
- Header text is now centered
- Cell size has been reduced for better aesthetic 

Trash Column:
- Cell size has been reduced for better aesthetic

### CODE REVIEW

**Comment by ajainarayanan** [comment]
@PGuimarais This is what I am seeing in my local.
screen shot 2016-06-09 at 1 10 53 pm

**Comment by ajainarayanan** [comment]
Also this is happening in 'Get Schema' in `GroupbyAggregate` plugin
screen shot 2016-06-09 at 1 12 11 pm

File: cdap-ui/app/features/hydratorplusplus/bottompanel.less [hunk 1]
```diff
@@ -318,16 +315,15 @@ body.theme-cdap.state-hydratorplusplus {
           padding-right: 20px;
         }
         &.name {
-          width: 35%;
+
```
Comment (elfenheart) [inline cdap-ui/app/features/hydratorplusplus/bottompanel.less]: remove empty rule

File: cdap-ui/app/directives/plugin-functions/functions/output-schema/output-schema.less [hunk 2]
```diff
@@ -23,3 +23,7 @@ output-schema {
     right: 0;
   }
 }
+
+.nullable {
+	width: 35px;
```
Comment (elfenheart) [inline cdap-ui/app/directives/plugin-functions/functions/output-schema/output-schema.less]: indentations

Comment (elfenheart) [inline cdap-ui/app/directives/plugin-functions/functions/output-schema/output-schema.less]: Indentations

**Comment by elfenheart** [comment]
get schema on database source. This is on plugin-functions > functions > get-schema

screen shot 2016-06-15 at 1 46 22 pm

**Comment by bdmogal** [comment]
good catch @elfenheart. How about Tracker?

**Comment by elfenheart** [comment]
@bdmogal  Tracker is fine, it's already according to the Tracker spec :)

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**